Source/WebCore/contentextensions/ContentExtensionParser.cpp

```diff
@@ -206,6 +207,7 @@ bool isValidCSSSelector(const String& selector)
     ASSERT(isMainThread());
     initializeCommonAtomStrings();
     QualifiedName::init();
+    ProcessWarming::initializeNames();
```

Comment:
You should call the right init functions directly instead of relying on ProcessWarming::initializeNames(). Per its name, it is meant to be used for process warming.

Reply:
ProcessWarming::initializeNames does what we need here.  Maybe we should rename that function

Reply:
It does more than what you need. Please don't rename.